Lib/BIOSLIB/CPUPARAM.LIB

```diff
@@ -162,14 +162,7 @@ END DESCRIPTION ***************************************************************/
 #asm xmem nodebug
 _enableClockModulation::
 	bool	hl
-#if _CPU_ID_ < R2000_R3
-	ld		l, h
-	lret
-#elif ENABLE_SPREADER == 0	// spreader disabled
-	xor	a
-	ld		(MTCRShadow), a
-;	ioi	ld (MTCR), a		// normal /OE0 and /OE1 timing
-	IOWRITE_A(MTCR)
+#if _CPU_ID_ < R2000_R3 || ENABLE_SPREADER == 0

```

Comment:
This change ensures that we don't disable "early /OE" if it was previously enabled.  Startup default is disabled (0x00), but the `StdBios.c` change around line 2455 will automatically enable it on all products with a doubled clock, and later code in this function enables it before turning on the spectrum spreader.